certora/specs/Health.spec

```diff
@@ -2,7 +2,10 @@
 methods {
     function extSloads(bytes32[]) external returns bytes32[] => NONDET DELETE;
 
+    function totalBorrowAssets(MorphoHarness.Id) external returns uint256 envfree;
     function totalBorrowShares(MorphoHarness.Id) external returns uint256 envfree;
+    function virtualTotalBorrowAssets(MorphoHarness.Id) external returns uint256 envfree;
+    function virtualTotalBorrowShares(MorphoHarness.Id) external returns uint256 envfree;
```

Comment:
```suggestion
    function totalBorrowShares(MorphoHarness.Id) external returns uint256 envfree;
```

Reply:
Those are actually needed in `StayHealthy.spec` now